hooks/useMultiStepForm.tsx

```diff
@@ -27,6 +32,6 @@ export function useMutistepForm(steps: FormStep[]) {
     next,
     back,
     isFristStep: currentStepIndex === 0,
```

Comment:
Typo, mentioned previously as well.

```suggestion
    isFirstStep: currentStepIndex === 0,
```

Reply:
My bad